### ISSUE TITLE
Listing a new app and existing apps after creating a new app(when a user clicks 'goBackBtn')

### ISSUE THREAD

**Issue by flyjwayur** [Rookies-RGSOC2016/Hoodie-electron-demo#25]

**Comment by flyjwayur** [comment]
To display the existing apps and new apps, I used 'apps.name'. But it only displays '-' as an app list. 
It also displays 'create-app-container and detail-app-container' together in index.html.

``` js
var goBackBtn = document.querySelector("#goBackBtn");
var appLists = document.querySelector("#appLists");
if(goBackBtn){
    goBackBtn.addEventListener("click",function(){
    apps.forEach(function(app){
            var $li = document.createElement('li');
            $li.innerHTML = `
                ${apps.name || '-'}
            `
            appLists.appendChild($li);
        });
        console.log('apps JSON.stringify:'+ apps.name);
        $("#apps-container,#create-app-container").toggle();
    });
}
```

**Comment by flyjwayur** [comment]
closed #25
